1 - DataTypes/Code/WeirdCalculator.cs

```diff
@@ -39,29 +44,44 @@ public static decimal ComplexDecimalAdd(float firstValue, double secondValue, de
 			 * I will give you one clue / tip - You *will* need to handle exceptions for the final solution.
 			 * If you are new to exceptions, google "C# Exception Handling" - litterally half our jobs is googling :P
 			 */
-
-			throw new NotImplementedException();
-		}
+            decimal result = 0.0M;
```

Comment:
As you are defining your type in the assigning part of this code, you can use `var`.

`var result = 0.0m`